### PR TITLE
WIP: Add the ceph serverspec

### PR DESCRIPTION
**Pull request by vlope093**
Dependent on PR #1554

### CODE REVIEW

File: roles/ceph-common/templates/etc/serverspec/ceph_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+         '',
+         '/var/log/ceph/*.log',
+         '{',
+         'daily',
+         'missingok',
+         'rotate 7',
+         'compress',
+         '}']
+  file_contents.each do |file_line|
+    it { should contain file_line}
```
Comment (pwplusnick) [inline roles/ceph-common/templates/etc/serverspec/ceph_spec.rb]: Fix whitespace

File: roles/ceph-common/tasks/serverspec.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+- name: serverspec checks for cinder
```
Comment (j2sol) [inline roles/ceph-common/tasks/serverspec.yml]: Why are the cinder checks happening in ceph? Copy/pasta I assume.

**Comment by j2sol** [comment]
Just two minor nits.

**Comment by j2sol** [comment]
ok to test

File: roles/ceph-common/tasks/main.yml [hunk 3]
```diff
@@ -90,3 +90,9 @@
     - logrotate
     - logging
   when: logging.enabled|default('True')|bool
+
+- include: severspec.yml
```
Comment (j2sol) [inline roles/ceph-common/tasks/main.yml]: s/severspec/serverspec/

**Comment by j2sol** [comment]
This is still broken, a typo.

**Pull request merged**